benchmarks/400.inference/421.image-captioning/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers==4.44.2
```

Comment:
With which Python versions did you test it?

Reply:
3.10